zanata-war/src/main/webapp/WEB-INF/layout/dashboard/projects.xhtml

```diff
@@ -6,14 +6,6 @@
   xmlns:a4j="http://richfaces.org/a4j"
   xmlns:zanata="http://java.sun.com/jsf/composite/zanata">
 
-  <a4j:jsFunction name="nextProjectPage"
-    render="projectListForm" action="#{dashboardAction.projectList.nextPage}"
-    status="projectlist-loader"/>
-
-  <a4j:jsFunction name="prevProjectPage"
-    render="projectListForm" action="#{dashboardAction.projectList.prevPage}"
-    status="projectlist-loader"/>
```

Comment:
So I was told, and I can't find any refs to either.

On 1 March 2016 at 10:25, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/main/webapp/WEB-INF/layout/dashboard/projects.xhtml
> <https://github.com/zanata/zanata-server/pull/1072#discussion_r54504153>:
>
> > @@ -6,14 +6,6 @@
> >    xmlns:a4j="http://richfaces.org/a4j"
> >    xmlns:zanata="http://java.sun.com/jsf/composite/zanata">
> >
> > -  <a4j:jsFunction name="nextProjectPage"
> > -    render="projectListForm" action="#{dashboardAction.projectList.nextPage}"
> > -    status="projectlist-loader"/>
> > -
> > -  <a4j:jsFunction name="prevProjectPage"
> > -    render="projectListForm" action="#{dashboardAction.projectList.prevPage}"
> > -    status="projectlist-loader"/>
>
> Just want to make sure these are indeed not needed anymore.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1072/files#r54504153>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat